cmd/generator/main.go

```diff
@@ -23,5 +24,11 @@ func main() {
 	if err != nil {
 		panic(fmt.Sprintf("cannot calculate the absolute path with %s", rootDir))
 	}
-	pipeline.Run(config.GetProvider(), absRootDir)
+	p := config.GetProvider()
+	pipeline.Run(p, absRootDir)
+	if fp := os.Getenv("SKIPPED_RESOURCES_CSV"); len(fp) != 0 {
+		if err := os.WriteFile(fp, []byte(strings.Join(p.GetSkippedResourceNames(), ",")), 0o600); err != nil {
```

Comment:
should these be using `;` so each resource is on its own row? looks like all the resources are generated into a single row now.  Not sure which is better, it's just the first thing I noticed when opening the csv is that you have to scroll horizontally forever because they are in one row 😇

Reply:
Replaced the separator char with `;`. @jbw976, which spreadsheet editor are you using? After seeing your comment, I gave it a try and it's opened by Numbers. With Numbers, both `,` and `;` result in a table with a single row. If we would prefer a table with a single column, then we may use `\n` as the separator char here.

Reply:
yes sir, Pages gets opened by default on my Mac.  When I mentioned `;`, I didn't try it out or think it through :)

My personal opinion is that whatever works to make it a single column instead of single row is a more friendly scrolling experience.  But that is not a super strong opinion and you can make the call here 🙇

Reply:
Then, let's proceed with a `\n` as the separator char to get a single column table.